common-template/src/main/java/com/redhat/cloud/notifications/qute/templates/extensions/SeverityExtension.java

```diff
@@ -0,0 +1,31 @@
+package com.redhat.cloud.notifications.qute.templates.extensions;
+
+import com.redhat.cloud.event.parser.ConsoleCloudEvent;
+import com.redhat.cloud.notifications.ingress.Action;
+import com.redhat.cloud.notifications.qute.templates.Severity;
+import io.quarkus.qute.TemplateExtension;
+
+public class SeverityExtension {
+
+    /** Cloud events do not support the severity field */
+    @TemplateExtension
+    static String severityAsEmailTitle(ConsoleCloudEvent event) {
+        return "";
+    }
+
+    @TemplateExtension
+    static String severityAsEmailTitle(Action action) {
+        Severity severity;
+        try {
+            severity = Severity.valueOf(action.getSeverity());
```

Comment:
Since we already extracted/checked/transformed the severity in engine, we could simplify this TemplateExtension as what you did previously in qute templates, like:
```
if (action.gerSeverity() == null 
     || "UNDEFINED".equals(action.getServerity()) 
     || "NONE".equals(action.getServerity())) {
      return "";
} else {
      return String.format("[%s] ", severity);
}
```